tools/ncf.py

```diff
@@ -428,8 +429,10 @@ def get_all_techniques_metadata(include_methods_calls = True, migrate_technique
     with codecs.open(file, encoding="utf-8") as fd:
       content = fd.read()
     try:
+      category = os.path.basename(os.path.dirname(os.path.dirname(os.path.dirname(file))))
```

Comment:
could you comment: "we need to go up 3 directories: technique version, name, and finaly category"